cmd/proxy.go

```diff
@@ -0,0 +1,57 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/gravitl/netclient/functions"
+	"github.com/spf13/cobra"
+)
+
+// proxyCmd represents the proxy command
+var proxyCmd = &cobra.Command{
+	Use:   "proxy [ on | off ]",
+	Short: "proxy on/off",
+	Long:  `switches proxy on/off`,
+
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) == 0 {
```

Comment:
this check is not required if you set Args; cobra.ExactArgs(1)  at line 14.... Let corbra do the work for you.